### PR TITLE
Add two torch examples

### PR DESCRIPTION
**Pull request by yiliu30**
## Type of Change

New Examples
Follow-up PR for https://github.com/intel/neural-compressor/pull/1670

- [x] Vision model
- [x] Mistrial-7b
- [x] Add these examples to weekly test @chensuyue 

New hello world example [page](https://github.com/intel/neural-compressor/tree/toy_examples/examples/helloworld)

### CODE REVIEW

**Review by XuehaoSun — CHANGES_REQUESTED** [review]

File: examples/README.md [hunk 1]
```diff
@@ -10,7 +10,8 @@ Intel® Neural Compressor validated examples with multiple compression technique
 * [Quick Get Started Notebook of Intel® Neural Compressor for Pytorch](/examples/notebook/pytorch/Quick_Started_Notebook_of_INC_for_Pytorch.ipynb)
 
 # Helloworld Examples
-
+* [torch_llm](/examples/helloworld/torch_llm): apply the weight-only quantization to LLMs.
+* [torch_non_llm](/examples/helloworld/torch_non_llm): apply the static quantization to non-LLMs.
```
Comment (Kaihui-intel) [inline examples/README.md:14]: How about the general quantization naming torch?

File: examples/helloworld/torch_llm/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+### Setting up the environment 
+```shell
+pip install -r requirements.txt
+```
+
+### Running the Quantization Script
+
+```shell
+python quant_mistral.py
```
Comment (Kaihui-intel) [inline examples/helloworld/torch_llm/README.md:14]: the script name should align with tf examples.

File: examples/helloworld/torch_non_llm/requirements.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
+neural-compressor>=2.3
+torch
+torchvision
```
[the remark this answers is not in the document]
Comment (yiliu30) [reply examples/helloworld/torch_non_llm/requirements.txt:3]: Oh, it should be a vision model that does not requires the transformers. Updated.

File: examples/README.md (hunk 1 above)
Comment (yiliu30) [reply examples/README.md:14]: Thanks, updated.

File: examples/helloworld/torch_llm/README.md (hunk 2 above)
Comment (yiliu30) [reply examples/helloworld/torch_llm/README.md]: It appears that `quant_mistral.py` is more meaningful than `test.py`. :)

**Review by XuehaoSun — APPROVED** [review]
test pass.
[test link](https://inteltf-jenk.sh.intel.com/job/lpot-feature-test-top/818/)

**Review by chensuyue — APPROVED** [review]

**Pull request merged**